examples/gallery/maps/country_polygons.py

```diff
@@ -0,0 +1,100 @@
+"""
+Highlight country and continent polygons
+----------------------------------------
+The :meth:`pygmt.Figure.coast` method can
+highlight country polygons via the ``dcw``
+parameter. It accepts the country code or
+full country name and can draw its borders
+and add a color to its landmass. It's also
+possible to define multiple countries at
+once by separating the indiviudal names
+with commas.
+"""
+
+# sphinx_gallery_thumbnail_number = 2
+
+
+import pygmt
+
+fig = pygmt.Figure()
+
+fig.coast(
+    region=[-12, 32, 34, 72],
+    projection="M6c",
+    land="gray",
+    water="white",
+    frame=True,
```

Comment:
Scandinavia looks super long on the Mercator projection. Could we try another projection? E.g. [Lambert Azimuthal Equal-Area projection](https://en.wikipedia.org/wiki/Lambert_azimuthal_equal-area_projection)? Basing this on [EPSG:3035](https://epsg.io/3035).

```suggestion
    # Lambert Azimuthal Equal Area lon0/lat0/horizon/width
    projection="A10/52/25/6c",
    land="gray",
    water="white",
    frame="afg",
```

![Lamber Azimuthal Equal-Area projection view of Europe.](https://github.com/GenericMappingTools/pygmt/assets/23487320/739aad9d-4570-43f6-93f4-cfdf026043a7)